src/www/status_wireless.php

```diff
@@ -187,9 +187,9 @@
 								</table>
 				</div>
 
-				<b>Flags:</b> A = authorized, E = Extended Rate (802.11g), P = Power save mode<br />
-	<b>Capabilities:</b> E = ESS (infrastructure mode), I = IBSS (ad-hoc mode), P = privacy (WEP/TKIP/AES),
-		S = Short preamble, s = Short slot time
+				<b><?=gettext('Flags:') ?></b> <?=gettext('A = authorized, E = Extended Rate (802.11g), P = Power save mode') ?><br />
+	<b><?=gettext('Capabilities:') ?></b> <?=gettext('E = ESS (infrastructure mode), I = IBSS (ad-hoc mode), P = privacy (WEP/TKIP/AES),
+		S = Short preamble, s = Short slot time') ?>
```

Comment:
Still not pretty, but it works a lot better than before and the strings are split very logical. Nice. :)